src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+:root {
```

Comment:
The `:root` selector is typically used for defining CSS variables. If there are no other global styles being set, this is fine, but ensure that you're not using it to define global styles that should be elsewhere.